### PR TITLE
test: fix component and module tests

### PR DESCRIPTION
**Pull request by vakrilov**

### CODE REVIEW

File: src/generate/module/index_spec.ts [hunk 1]
```diff
@@ -1,55 +1,49 @@
...
-  beforeEach(() => {
-    appTree = new VirtualTree();
-    appTree = createAppModule(appTree);
-    appTree = createEmptyProject(appTree);
-  });
- 
+  let appTree: UnitTestTree;
+
+
+  // Structure proposal
```
Comment (sis0k0) [inline src/generate/module/index_spec.ts]: You can drop this comment.

File: src/generate/module/index_spec.ts [hunk 2]
```diff
@@ -1,55 +1,49 @@
...
   describe('when in ns-only project', () => {
+    beforeEach(() => {
+      appTree = createEmptyNsOnlyProject(project);
+    });
+
+    // tests
```
Comment (sis0k0) [inline src/generate/module/index_spec.ts]: This one as well.

File: src/generate/utils.ts [hunk 3]
```diff
@@ -36,10 +36,9 @@ const isNs = (tree: Tree) => {
   return !!packageJson.nativescript;
 }
 
-// TODO: need a better way to identify {A} Web projects
 const isWeb = (tree: Tree) => {
   if (!tree.exists('nsconfig.json')) {
-    console.log(`nsconfig.json not found. Assuming this is a {N} only project`);
+    // console.log(`nsconfig.json not found. Assuming this is a {N} only project`);
```
Comment (sis0k0) [inline src/generate/utils.ts]: Should we get this one back or remove it?

File: src/generate/module/index_spec.ts (hunk 2 above)
Comment (vakrilov) [reply src/generate/module/index_spec.ts]: Removed! Perfectly descriptive comment though :trollface:

**Pull request merged**